trackpy/plots.py

```diff
@@ -260,6 +260,7 @@ def pairwise(iterable):
         _plot_style.update(markeredgecolor=color[-1])
         ax.plot(centroids['x'][high], centroids['y'][high],
                 **_plot_style)
+    ax.invert_yaxis()
```

Comment:
95% sure, sometime the auto-scaling surprises me, but once the limits are set manually, the auto-scaling should be turned off.